packages/web-app-files/src/store/actions.ts

```diff
@@ -200,7 +200,7 @@ export default {
 
             translated = $gettext('Failed to delete "%{file}" - the file is locked')
           }
-          const title = $gettextInterpolate(translated, { file: file.name }, true)
+          const title = $gettext(translated, { file: file.name }, true)
```

Comment:
You can remove `$gettext` from `translated`, otherwise you're wrapping multiple `$gettext` calls.